modules/nw-operator-vsphere-multiple-subnets.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/upi/installing-vsphere-network-customizations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-operator-vsphere-multiple-subnets_{context}"]
+= Specifying multiple subnets for your network
+
+Before you install an {product-title} cluster on a {vmw-short} host, you can specify multiple subnets for a networking implementation so that the {vmw-short} cloud controller manager (CCM) can select the appropriate subnet for a given networking situation. The {vmw-full} can use the subnet for managing pods and services on your cluster. For this configuration, you must specify internal and external Classless Inter-Domain Routing (CIDR) implementations in the {vmw-short} cloud controller manager (CCM) configuration. Each CIDR implementation lists an IP address range that the CCM uses to decide what subnets interact with traffic from internal and external networks.
+
+If you do not configure internal and external CIDR implementations in the {vmw-short} CCM configuration, the {vmw-short} CCM selects the first subnet that the host operating system reports as the primary network interface. This configuration can cause new nodes that associate with a `MachineSet` object with a single subnet to become unusable as each new node receives the `node.cloudprovider.kubernetes.io/uninitialized` taint. In certain situations, the reported network interface might not be the expected network interface. These situations can cause communication issues with the Kubernetes API server that can cause installation of the cluster to fail.
```

Comment:
This is a little weird now because I'm not sure what the behavior is in a node with a single NIC and multiple subnets. Does the CCM just pick the first subnet? I would assume so, but I don't know.

Maybe we just drop the specific explanation of how it works and say that if you do not configure these values the wrong subnet may be selected and then keep the discussion of the uninitialized taint? Maybe remove the "In certain situations, the reported network interface might not be the expected network interface." line too since it doesn't entirely make sense in a single NIC scenario either, and I think the important points about the uninitialized taint causing deployment failure are still there.